index.html

```diff
@@ -17,8 +17,16 @@
     <header class="toolbar">
         <a href="#" class="logo">My Logo</a>
         <nav class="fas fa-bars"></nav>
+        <div class="menu">
+            <div class="fa fa-times" aria-hidden="true"></div>
+            <ul class="menu-list">
+                <li><a href="#portfolio" style="text-decoration: none;">Portfolio</a></li>
+                <li><a href="#about" style="text-decoration: none;">About</a></li>
+                <li><a href="#contact" style="text-decoration: none;">Contact</a></li>
+            </ul>
+        </div>
```

Comment:
- [ ] As stated in the Project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), Kindly Adjust the space between mobile menu options and margin of close(X) button Just Like The Figma design .
The Figma design Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/194937387-8bef269b-993a-425f-89e8-787ca7034e2b.PNG)

Your Project design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/194937654-12b0ff74-a551-46d4-81bd-7660303e6b6b.PNG)